test/sanity/validate-modules/validate-modules

```diff
@@ -203,6 +203,7 @@ class ModuleValidator(Validator):
         'async_status.ps1',
         'slurp.ps1',
         'setup.ps1'
+        'gather_facts.ps1'
```

Comment:
This line should replace the previous line. Include a comma at the end of the line.